test_red_black_cards.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from red_black_cards import expected_value
+
+class testGame(unittest.TestCase):
```

Comment:
We need more test cases:

- What if `b` or `r` is `<0`?
- What if they are `float`s or some invalid input? (You'll need to change your `expected_value` function to handle this
- We need more nontrivial test cases. What about `2,2` and `3,3`?